test/JuMPExtension.jl

```diff
@@ -225,6 +225,14 @@ function JuMP.add_constraint(model::MyModel, c::JuMP.AbstractConstraint,
     JuMP.set_name(cref, name)
     return cref
 end
+
+function JuMP.dual_start_value(constraint_ref::MyConstraintRef)::Union{Nothing, Float64}
+    return dual_start_value(constraint_ref)
+end
+function JuMP.set_dual_start_value(constraint_ref::MyConstraintRef, value::Float64)
+    set_dual_start_value(constraint_ref, value)
```

Comment:
I'm not sure how to edit this method. Should be similar to https://github.com/JuliaOpt/JuMP.jl/blob/master/test/JuMPExtension.jl#L163 right?

Reply:
Easiest thing is to probably not implement these methods, move the `@testset` to inside this testset 
https://github.com/JuliaOpt/JuMP.jl/blob/4ffcd1ff2423f69718d1ab18e3af882594bc75e7/test/constraint.jl#L484, and use `model = Model()` instead of `model = ModelType()`.

Reply:
Thanks. works now :v: